velox/functions/prestosql/aggregates/ArbitraryAggregate.cpp

```diff
@@ -298,8 +298,13 @@ bool registerArbitraryAggregate(const std::string& name) {
       });
 }
 
-static bool FB_ANONYMOUS_VARIABLE(g_AggregateFunction) =
-    registerArbitraryAggregate(kArbitrary);
-
 } // namespace
+
+namespace prestosql {
+
+void registerArbitraryAggregates() {
```

Comment:
I find it confusing to use plural for singular. Maybe you can solve the problem you mentioned by moving anonymous namespace inside prestosql. That's a pattern used throughout the codebase.

Reply:
> I find it confusing to use plural for singular. Maybe you can solve the problem you mentioned by moving anonymous namespace inside prestosql. That's a pattern used throughout the codebase.

I will try it.